cluster/juju/layers/kubernetes-master/templates/addons/dashboard-controller.yaml

```diff
@@ -0,0 +1 @@
+# Empty placeholder template, to be replaced at build time
```

Comment:
Yeah, we need to just have those files added to the manifest when the tactic runs, which is possible. @johnsca might be able to shed some light